ajax/libs/covervid/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "covervid",
+  "filename": "covervid.min.js",
+  "version": "1.0.0",
+  "description": "Make your HTML5 video behave like a background cover image with this lightweight Javascript plugin extension",
+  "author": "Stefan Erickson",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stefanerickson/covervid"
+  },
+  "keywords": [
+    "video",
+    "image",
+    "png"
+  ],
+  "npmName": "covervid",
```

Comment:
I think https://www.npmjs.com/package/covervid is not officially maintained. We cannot use it.

There is no git tags in the GitHub repo. I think we can add it without apply auto-update config.